### PR TITLE
EZ-393 Integrating alert component

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

File: templates/block/block--inline-block--alert-banner.html.twig [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+{% set data = {
+  "alert": {
+		"bgColor": content['#block_content'].field_background_color.0.color,
+		"content": content['#block_content'].body.0.value
+	}
+} %}
+
+<section{{ attributes.addClass(classes) }}>
+  {{ title_prefix }}
```
Comment (kuldev) [inline templates/block/block--inline-block--alert-banner.html.twig:67]: do we have this "h2" element on alert, if not lets remove this section entirely.  @surbhi90

Comment (surbhi90) [reply templates/block/block--inline-block--alert-banner.html.twig]: Removed "h2" element as it is not required

File: templates/block/block--inline-block--alert-banner.html.twig [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    'clearfix',
+  ]
+%}
+
+{% set data = {
+  "alert": {
+		"bgColor": content['#block_content'].field_background_color.0.color,
+		"content": content['#block_content'].body.0.value
+	}
+} %}
```
Comment (vaibhavjain-in) [inline templates/block/block--inline-block--alert-banner.html.twig:64]:
```suggestion
{% 
  set data = {
    "alert": {
	  "bgColor": content['#block_content'].field_background_color.0.color,
	  "content": content['#block_content'].body.0.value
	}
  } 
%}
```
Line 61, 62 and 63 are tabs than spaces. Lets fix those.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@surbhi90 Please see a minor feedback.

File: templates/block/block--inline-block--alert-banner.html.twig (hunk 2 above)
Comment (surbhi90) [reply templates/block/block--inline-block--alert-banner.html.twig]: Done

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**